frontend/src/metabase/notifications/NotificationsActionsMenu/test/setup.tsx

```diff
@@ -185,19 +173,42 @@ export function setupQuestionSharingMenu({
   setupAlertsEndpoints(card, alerts);
   setupUsersEndpoints([state.currentUser] as UserListResult[]);
 
+  setupNotificationChannelsEndpoints({
+    slack: { configured: isSlackSetup },
+    email: { configured: isEmailSetup },
+  } as any);
+
   if (isEnterprise) {
     setupEnterprisePlugins();
   }
 
   renderWithProviders(
     <div>
-      <QuestionSharingMenu question={new Question(card)} />
+      <QuestionNotificationsMenu question={new Question(card)} />
       <FakeSidebar />
     </div>,
     { storeInitialState: state },
   );
+
+  return {
+    cardId: card.id,
+  };
 }
 
-export const openMenu = () => {
-  return userEvent.click(screen.getByTestId("sharing-menu-button"));
+export const waitForChannelsConfigLoaded = async () => {
+  await waitFor(() => {
+    expect(fetchMock.called(`path:/api/pulse/form_input`)).toBe(true);
+  });
+
+  await delay(50);
```

Comment:
nit: wondering if there is a way to wait for this without using delays, e.g. waiting on a loader element